openapi/src/parts/admin.yaml

```diff
@@ -0,0 +1,36 @@
+## admin
+paths:
+  /api/admin/v1/workspaces/cloudcontexts/syncIamRoles:
+    patch:
+      summary: Sync iam roles of all workspaces.
+      operationId: syncIamRoles
+      tags: [Admin]
+      responses:
+        '200':
+          $ref: '#/components/responses/SyncIamRolesResultResponse'
```

Comment:
ah right - we don't really need a result endpoint for this

Reply:
can you clarify?

you would call this endpoint, get a jobid. and then calls the jobs endpoint to check for completion

Reply:
The shared job endpoint gives job status in a `JobReport` object, but doesn't provide a way to return an endpoint-specific result object. For other async endpoints, we provide one endpoint to start the job (like `Workspace/createCloudContext`) and then another endpoint to poll/fetch the result object (like `Workspace/getCreateCloudContextResult`).

I don't see any step in this flight setting output (via `FlightUtils.setResponse`) - since this is an admin only endpoint, it seems like we're just expecting the caller to look at the service logs for results instead.